tools/vscode_sql_extraction/.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 .vscode-test/
 *.vsix
 /resources/sql_extraction
+!/.vscode/
```

Comment:
Are all those files under the .vscode directory needed to build the plugin?

Reply:
I believe so. The samples on GitHub all have those 4 files.